tests/src/ExistingSiteJavascript/DrupalBlockComponentTest.php

```diff
@@ -28,15 +28,17 @@ class DrupalBlockComponentTest extends CohesionComponentTestBase {
    *
    * @dataProvider providerAddComponentToLayoutCanvas
    */
-  public function testComponent(array $roles = []) {
+  public function testComponent(array $roles = []): void {
     $account = $this->createUser();
     array_walk($roles, [$account, 'addRole']);
     $account->save();
     $this->drupalLogin($account);
 
     $this->drupalGet('/node/add/page');
+    /** @var \Drupal\FunctionalJavascriptTests\JSWebAssert $assert_session */
     $assert_session = $this->assertSession();
```

Comment:
@apathak18 please use same variable name for assert sessions - i.e $assertSession

cc @vishalkhode1

Reply:
I've follow the same variable name pattern in a method @deepakmishra2 so not required this change